### ISSUE TITLE
Missing shared libraries

### ISSUE THREAD

**Issue by qbitroot** [electron/electron-quick-start#486]
When I run `npm start`, it shows the following output:
```
> electron-quick-start@1.0.0 start /tmp/electron-quick-start
> electron .

/tmp/electron-quick-start/node_modules/electron/dist/electron: error while loading shared libraries: libcups.so.2: cannot open shared object file: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2021-05-20T16_43_01_585Z-debug.log
```
As 4th line says, there is no `libcups` shared library. I tried installing it, but there are new missing shared libraries. I don't use systemd and it may be one of the reasons of bug.

After hours of troubleshooting, I solved it by putting `.so` libraries from Debian 10 image into the `dist` directory. You can download them from [the attachment](https://github.com/electron/electron-quick-start/files/6517564/libs.zip) (for testing purposes).

**Comment by DrockDaniel** [comment]
I've never used linxsis

**Comment by DrockDaniel** [comment]
Sorry I'm pooped out.

**Comment by fullpl4nch3** [comment]
im using wsl 2 on windows 11, and im getting the same error. #519 point to this issue. So im kinda stuck in the loop, any ideas?

The error:

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/fullpl4nch3/.npm/_logs/2022-01-24T12_57_29_222Z-debug.log

**Comment by genru** [comment]
@qbitroot @fullpl4nch3  if use wsl, please try 
```
rm -rdf node_modules
npm install --platform=win32
```
to tell dependenies your are running on a win system. this resolved my problem.

**Comment by burt23** [comment]
Unfortunately the above command from @genru didn't work for me. Trying to run the overmind dev tools via npx and am running into this same error. Any help greatly appreciated, thank you in advance

**Comment by tammaroivan** [comment]
Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

**Comment by PoweredByEnergyDrinks** [comment]
> sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev

Thanks @tammaroivan , you legend!

**Comment by giuliohome** [comment]
Nope, having this issue today in Selenium IDE and I solved it by installing google-chrome on ubuntu on Windows WSL and using vcxsrv as Windows X Server with export DISPLAY ... in .bashrc ...

**Comment by anbarasu97** [comment]
> Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

This worked!!!

**Comment by LuigiImVector** [comment]
> Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

I love you

**Comment by giuliohome** [comment]
Ok, better to unsuscribe this thread

**Comment by long9nt** [comment]
> Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

awesome

**Comment by danielwust99** [comment]
> Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

Thanks, worked for me bro!

**Comment by ValsiRod** [comment]
I ran the same command 
`sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`
and got this error:
`error while loading shared libraries: libasound.so.2: cannot open shared object file: No such file or directory`

what else could I try?

**Comment by danielwust99** [comment]
> I ran the same command `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev` and got this error: `error while loading shared libraries: libasound.so.2: cannot open shared object file: No such file or directory`
> 
> what else could I try?

Try install the apel release packages before run.

**Comment by TeqGin** [comment]
> I ran the same command `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev` and got this error: `error while loading shared libraries: libasound.so.2: cannot open shared object file: No such file or directory`
> 
> what else could I try?

`sudo apt-get install libasound2`

this works for me

**Comment by marcospgp** [comment]
There is this closed issue about documenting runtime (as opposed to build time) dependencies for Electron: https://github.com/electron/electron/issues/26673

**Comment by manojborugadda** [comment]
Thank you so much @tammaroivan  💯

**Comment by Carolzhangzz** [comment]
> Could you try with `sudo apt install libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev`, that worked for me

thanks ! that works
